src/core/grid_based_algorithms/lb_interface.cpp

```diff
@@ -1052,10 +1052,7 @@ double lb_lbnode_get_density(const Utils::Vector3i &ind) {
 #ifdef CUDA
     int single_nodeindex = ind[0] + ind[1] * lbpar_gpu.dim_x +
                            ind[2] * lbpar_gpu.dim_x * lbpar_gpu.dim_y;
-    static std::vector<LB_rho_v_pi_gpu> host_print_values(0);
-
-    if (host_print_values.empty())
-      host_print_values.resize(1);
+    static std::array<LB_rho_v_pi_gpu, 1> host_print_values;
```

Comment:
array of size 1 looks suspicious. Why is this an array?

Reply:
True. It was an array before, I just kept the memory access like it was. Removed the array now.